frontend/src/Feed.js

```diff
@@ -9,6 +9,7 @@ class Feed extends React.Component {
 
         // This is a stateful component so we need to define a default representation of our state
         this.state = {
+            text: '',
```

Comment:
We set up the default state here. Its been updated to include an empty text field.